polytracker/scripts/polybuild.py

```diff
@@ -149,14 +155,18 @@ def instrument_bitcode(bitcode_file: Path, output_bc: Path,
     subprocess.check_call(opt_command)
     if ignore_lists is None:
         ignore_lists = []
+    opt_command = ["opt", "-enable-new-pm=0", "-load", str(META_PASS_PATH), "-meta", str(bitcode_file), "-o", str(bitcode_file)]
+    ret = subprocess.call(opt_command)
+    if ret != 0:
+        print("Metadata pass failed! Exiting!")
```

Comment:
Better to also print the command itself so the user can debug it:
```suggestion
        print(f"Metadata pass exited with code {ret}:\n{' '.join(opt_command)}")
```